[2025-09-09T21:40:15.321+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-09-09T21:40:15.353+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag2.v2.0.hello_user scheduled__2025-09-02T03:00:00+00:00 [queued]>
[2025-09-09T21:40:15.368+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag2.v2.0.hello_user scheduled__2025-09-02T03:00:00+00:00 [queued]>
[2025-09-09T21:40:15.381+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-09-09T21:40:15.413+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): hello_user> on 2025-09-02 03:00:00+00:00
[2025-09-09T21:40:15.428+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag2.v2.0', 'hello_user', 'scheduled__2025-09-02T03:00:00+00:00', '--job-id', '99', '--raw', '--subdir', 'DAGS_FOLDER/dag2.py', '--cfg-path', '/tmp/tmp300lrt0e']
[2025-09-09T21:40:15.436+0000] {standard_task_runner.py:105} INFO - Job 99: Subtask hello_user
[2025-09-09T21:40:15.440+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=203) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-09-09T21:40:15.441+0000] {standard_task_runner.py:72} INFO - Started process 217 to run task
[2025-09-09T21:40:17.765+0000] {task_command.py:467} INFO - Running <TaskInstance: dag2.v2.0.hello_user scheduled__2025-09-02T03:00:00+00:00 [running]> on host 4461bcf72552
[2025-09-09T21:40:15.977+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag2.v2.0' AIRFLOW_CTX_TASK_ID='hello_user' AIRFLOW_CTX_EXECUTION_DATE='2025-09-02T03:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-02T03:00:00+00:00'
[2025-09-09T21:40:15.980+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-09-09T21:40:16.062+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-09-09T21:40:16.082+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-09-09T21:40:16.083+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag2.v2.0, task_id=hello_user, run_id=scheduled__2025-09-02T03:00:00+00:00, execution_date=20250902T030000, start_date=20250909T214015, end_date=20250909T214016
[2025-09-09T21:40:16.149+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-09-09T21:40:16.198+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-09-09T21:40:16.203+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
